app/controllers/Analyse.scala

```diff
@@ -74,12 +77,35 @@ final class Analyse(
                 )
               )
               .flatMap: data =>
-                Ok.page(
-                  views.analyse.replay(
+                val pgn_without_clocks = pgn.tree match
```

Comment:
We follow the [scala naming conventions](https://docs.scala-lang.org/style/naming-conventions.html) that forbid use of snake_case